next.config.js

```diff
@@ -30,3 +31,40 @@ const nextConfig = {
 }
 
 module.exports = nextTranslate(nextConfig)
+
+// Injected content via Sentry wizard below
+
+const { withSentryConfig } = require('@sentry/nextjs')
```

Comment:
Please place this line in the beginning of this file, it is a best practice to have all required imports in the start of the file.